# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/nfs-utils/hotfix-unistd.patch.musl
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- nfs-utils-2.6.4/support/reexport/reexport.c.vanilla	2024-03-19 13:41:05.701340658 +0100
+++ nfs-utils-2.6.4/support/reexport/reexport.c	2024-03-19 13:41:14.076340236 +0100
@@ -8,6 +8,7 @@
 #include <sys/types.h>
 #include <sys/vfs.h>
 #include <errno.h>
+#include <unistd.h>
 
 #include "nfsd_path.h"
 #include "conffile.h"
--- nfs-utils-2.6.4/support/reexport/fsidd.c.vanilla	2024-03-19 13:40:17.128343105 +0100
+++ nfs-utils-2.6.4/support/reexport/fsidd.c	2024-03-19 13:41:22.888339792 +0100
@@ -7,6 +7,7 @@
 #include <dlfcn.h>
 #endif
 #include <event2/event.h>
+#include <unistd.h>
 
 #include "conffile.h"
 #include "reexport_backend.h"
